### PR TITLE
update 3rd party dependencies

### PR DESCRIPTION
**Pull request by atomfrede**
Updating some dependencies. Should we remove the `vendor.css.ejs` as it is not used anymore, right?

We might have a problem with the dummy data or with instant/zoned date time fields in general. They seem to be a string instead of date. Will try to investigate a little further. The result is you can't edit entities with fake data for instant/zoned date time fields. Does someone know the current behavior?

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-vuejs/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by atomfrede** [comment]
Maybe I was a bit to confident :( Will look into it. Looks like a problem with updated vue filters and changes in bootstrap vue. Manual tests are good though

**Comment by atomfrede** [comment]
So good news, there are some e2e tests failing (`.btn-danger` not found for example) so this might be problem with updated vue-bootstrap.

**Comment by sahbi-ktifa** [comment]
Sounds great, good job! Hope the CI will pass!

**Comment by atomfrede** [comment]
After I rebased the latest e2e improvements it looks much better and one test already passes. The remaining tests seem to be fixable, I guess due to some bootstrap changes the assertions are not correct anymore. Will investigate further

**Comment by vishal423** [comment]
@atomfrede, can you rebase against master to see if `.modal` related failed tests pass? I have updated po in recent changes

**Comment by atomfrede** [comment]
:+1: Will do this evening.

**Comment by atomfrede** [comment]
I have found the problem. It relates to https://bootstrap-vue.js.org/docs/components/modal#lazy-loading-and-static-modals. Setting the modal property to static restores the old behavior, but I think lazy loading the content sounds like a good idea. Will try to adapt the e2e spec, but currently no idea how to do it to be honest.

**Comment by vishal423** [comment]
Can you try replacing ```expect(await deletePage.modal.isDisplayed()).to.be.false;``` by ```expect(await isVisible(deletePage.modal)).to.be.false;``` in delete user test case?

**Comment by atomfrede** [comment]
Will try. For the modal itself I have solution. Now the ok button is not found

**Comment by vishal423** [comment]
Can you also check the difference with entity delete tests that pass?

File: generators/client/templates/vue/src/test/javascript/e2e/modules/account/account.spec.ts.ejs [hunk 1]
```diff
@@ -393,11 +393,13 @@ describe('Account', () => {
         userManagementPage = await navBarPage.getUserManagementPage();
 
         const deletePage: UserManagementDeletePage = await userManagementPage.getDeletePage('user_test');
+        await waitUntilDisplayed(deletePage.modal);
```
Comment (vishal423) [inline generators/client/templates/vue/src/test/javascript/e2e/modules/account/account.spec.ts.ejs]: I think this is not required as `getDeletePage()` call waits for an element `okButton` to be clickable

Comment (atomfrede) [reply generators/client/templates/vue/src/test/javascript/e2e/modules/account/account.spec.ts.ejs]: This is now the same as the entity spec. Without that call the spec failed with the ok button not clickable, although the fake user was deleted, but can try without that again locally.

**Pull request merged**